design/high-level.md

```diff
@@ -4,6 +4,6 @@
 
 At a very high level, a viewer goes to the website, the website fetches data from the API, and then if they watch a stream the website also fetches the stream from the video edge.
 
-When a streamer goes live they hit the video infrastructure and then there are some API calls, and their stream is pushed to video edge where the viewers can pick it up and watch the stream.
+When a Streamer goes live, they hit the video infrastructure, and then some API Calls are pushed by them to the video edge, where the viewers can pick it up and watch the stream
```

Comment:
Completely butchers the intent of "API calls --> stream is pushed to video edge". At least "and then some API calls push their stream to the video edge". Even better though "and after some API calls their stream is pushed to the video edge"

Reply:
`API calls are made` and `then the stream is pushed to video edge`.